scripts/uninstall.sh

```diff
@@ -31,6 +31,9 @@ remove_envs(){
 
   echo "Removing concoct environment..."
   conda env remove --name metabiome-concoct
+
+  echo "Removing concoct environment..."
```

Comment:
Correct environment name.
```suggestion
  echo "Removing DAS Tool environment..."
```